src/indices.py

```diff
@@ -2177,6 +2177,32 @@ class SpectralIndices(BaseModel):
             application_domain="vegetation",
             date_of_addition="2024-05-16",
             contributor="https://github.com/davemlz",
```

Comment:
Closing parenthesis is missing ;)

```suggestion
            contributor="https://github.com/davemlz",
        )
```

Reply:
Ups, forgot a comma:

```suggestion
            contributor="https://github.com/davemlz",
        ),
```